packages/experience/src/types/guard.ts

```diff
@@ -97,3 +104,92 @@ export const backupCodeErrorDataGuard = s.object({
 export const backupCodeBindingStateGuard = backupCodeErrorDataGuard;
 
 export type BackupCodeBindingState = s.Infer<typeof backupCodeBindingStateGuard>;
+
+/* WebAuthn */
+
+const webAuthnTransportGuard: s.Describe<WebAuthnTransport> = s.enums([
+  'usb',
+  'nfc',
+  'ble',
+  'internal',
+  'cable',
+  'hybrid',
+  'smart-card',
+]);
+
+const webAuthnRegistrationOptionsGuard: s.Describe<WebAuthnRegistrationOptions> = s.object({
```

Comment:
need to check whether we need to guard these detailed configs at the client side